omnibus/config/software/stackstate-agent-integrations.rb

```diff
@@ -153,12 +148,12 @@
 
     # Install the static environment requirements that the Agent and all checks will use
     if windows?
-      command("#{python_bin} -m #{python_pip_no_deps}\\datadog_checks_base")
-      #[VS] command("#{python_bin} -m #{python_pip_no_deps}\\datadog_checks_downloader --install-option=\"--install-scripts=#{windows_safe_path(install_dir)}/bin\"")
+      command("#{python_bin} -m #{python_pip_no_deps}\\stackstate_checks_base")
+      #[VS] command("#{python_bin} -m #{python_pip_no_deps}\\stackstate_checks_downloader --install-option=\"--install-scripts=#{windows_safe_path(install_dir)}/bin\"")
```

Comment:
ditch this

Reply:
discuss with mark downloading/security

Reply:
we discussed that for now we ship checks with the agent, and deb/rpm signing gives enough security